doc/rst/language/spec/conversions.rst

```diff
@@ -295,9 +295,8 @@ when one type is a subtype of another.
    *Example (subtype-three.chpl)*.
 
    This example demonstrates a combination of all three rules. Note that
-   ``ParentClass`` is a generic type meaning the class type
-   ``ParentClass`` with any memory management strategy
-   (:ref:`Class_Types`).
+   ``ParentClass`` is a generic type because it has generic memory
```

Comment:
Maybe "uses a generic memory management strategy"?  It seems like there's a word missing between "has" and "generic"

Reply:
Is "Note that ``ParentClass`` indicates a generic memory management strategy. (:ref:`Class_Types`)." OK?